backend/homework/5/src/main/java/org/example/q1/Main.java

```diff
@@ -0,0 +1,23 @@
+package org.example.q1;
+
+public class Main {
+    public static void main(String[] args) {
+        MessageQueue messageQueue = new MessageQueue();
+
+        Thread sender1 = new Thread(new MessageSender(messageQueue, "Sender1"));
+        Thread sender2 = new Thread(new MessageSender(messageQueue, "Sender2"));
+        Thread sender3 = new Thread(new MessageSender(messageQueue, "Sender3"));
```

Comment:
even though its a small usecase but you could have extracted it into methods returning new thread as we are duplicating same thing 3 times. `private static Thread getMessageSenderThread() and getMessageReceiverThread`